CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 Mapbox welcomes participation and contributions from everyone.
 
 # main
+## Dependencies
+* Update common to v23.3.0. ([1984](https://github.com/mapbox/mapbox-maps-android/pull/1984))
```

Comment:
Should we add that the known issue (NPE if last location not available) from rc.1 is fixed?

Reply:
Yes, will do that when I will bump gl-native and copy both native and common changelogs.